test/e2e_node/services/services.go

```diff
@@ -64,15 +64,19 @@ func NewE2EServices(monitorParent bool) *E2EServices {
 // standard kubelet launcher)
 func (e *E2EServices) Start() error {
 	var err error
+	if e.services, err = e.startInternalServices(); err != nil {
```

Comment:
At this point, the Kubelet is already started without a Bearer Token in the configuration for conformance, maybe the token must be generated on ```test/e2e_node/services/services.go``` and passed down the stack.

Reply:
Yes, I think something like this is the case. I am taking a break from this for the moment, and have not started looking at the details. If someone else wants to keep looking for today, I'm happy to let them continue, otherwise I'll start again tomorrow.

Reply:
Running remote with the PR, This is the Kubeconfig:

```
@test-cos-beta-85-13310-1041-1 /tmp/node-e2e-20201103T122457 $ cat kubeconfig
apiVersion: v1
kind: Config
users:
- name: kubelet
  user:
    token: loiXGCHWaBpe9m94
clusters:
- cluster:
    server: https://127.0.0.1:6443
    insecure-skip-tls-verify: true
  name: local
contexts:
- context:
    cluster: local
    user: kubelet
  name: local-context
```

This is the run-services-mode rerun:
```
/usr/local/bin/e2e_node.test --run-services-mode --bearer-token=mjBCSf8Jd_N0FSTb ... 
```

Still breaking on both API server and Kubelet having a wrong token:

```
E1103 12:31:43.827172     103 authentication.go:53] Unable to authenticate the request due to an error: invalid bearer token 
```

Reply:
hmm I guess this has something to do with how ginkgo run the tests https://github.com/kubernetes/kubernetes/pull/94723#issuecomment-700512458, I would take a deeper look, thanks for pointing it out.

Reply:
yes, when started, we're outside of the ginkgo runner.